File: iftest/iftest.php
<?php

$t1 = true;
$t2 = 1;
$t3 = 0;

/*echo "Start use if = 1\n====\n";
for($x = 0 ;$x<10;$x++){
$t3=0;
$time=microtime(true);
for($i = 0 ;$i < 100000000;$i++){
	if($t2 == 1){
		$t3++;
	}
}
$time = round(microtime(true) - $time,8);

echo "\tUse $time s\n";
}
echo "Stop use if = 1\n";*/
echo "Start use if true\n====\n";

for($x = 0 ;$x<10;$x++){
$t3=0;
$time=microtime(true);
for($i = 0 ;$i < 100000000;$i++){
	if($t1){
		$t3++;
	}
}
$time = round(microtime(true) - $time,8);

echo "\tUse $time s\n";
}


?>

File: iftest/iftest.c
#include "stdio.h"
int main(){
	int t1 = 1,x=0,i=0;
	printf("Start use if = 1\n====\n");
//	for(x = 0 ;x<10;x++){
		int t3=0;
		for(i = 0 ;i < 100000000;i++){
	    	if(t1 ==1){
    	    	t3++;
	    	}
		}

		printf( "\tUse $time s\n");
//	}
//	printf("Stop use if = 1\n");

	return 0;
}
